src/main/java/org/team4159/frc/robot/Constants.java

```diff
@@ -98,6 +100,12 @@
   }
 
   public static final class SHOOTER_CONSTANTS {
+    public static final int COUNTS_PER_SECOND_TO_RPM = ENCODERS.THROUGH_BORE_ENCODER_CPR * 60;
```

Comment:
use `Gearing` class?

Reply:
i dont really understand the math, what would the gear ratio be